client/posts/posting/image.ts

```diff
@@ -0,0 +1,32 @@
+// Reload image if error
+// Handle event if image error
+async function onImageErr(e: Event) {
+    e.stopPropagation()
+    e.preventDefault()
+    const el = (e.target as HTMLImageElement)
+
+    if (el.tagName !== "IMG" || (el.complete && el.naturalWidth !== 0)) {
+        return
+    }
+
+    const src = el.getAttribute("src")
+
+    if (src.includes("?bs=")) {
+        return
+    }
+
+    for (var i = 0;; i++) {
```

Comment:
Note: I tried limiting it, but for some reason everything behaves weirdly in async functions and it would just skip over, or not actually do the await or something.